specification/Metadata/README.md

```diff
@@ -0,0 +1,446 @@
+> |:-----------------|:---------------------------|:-------:|:-------------:|---------------:|
+> | forwardDirection | "Forward direction vector" | "VEC3"  |   "FLOAT64"   |              3 |
+> | passengers       | "Passenger names"          | "ARRAY" |   "STRING"    |                |
+> | modelMatrix      | "4x4 model matrix"         | "MAT4"  |   "FLOAT32"   |             16 |
+
+#### Component Type
+
+Properties may be comprised of one component (`SINGLE`) or many components (`ARRAY`, `VECN`, `MATN`), depending on the property `type`. Each component is an instance of the property's component type (`componentType`), with the following component types supported:
+
+| name    | componentType                                                             |
+|---------|---------------------------------------------------------------------------|
+| INT8    | Signed integer in the range `[-128, 127]`                                 |
+| UINT8   | Unsigned integer in the range `[0, 255]`                                  |
+| INT16   | Signed integer in the range `[-32768, 32767]`                             |
+| UINT16  | Unsigned integer in the range `[0, 65535]`                                |
+| INT32   | Signed integer in the range `[-2147483648, 2147483647]`                   |
+| UINT32  | Unsigned integer in the range `[0, 4294967295]`                           |
+| INT64   | Signed integer in the range `[-9223372036854775808, 9223372036854775807]` |
+| UINT64  | Unsigned integer in the range `[0, 18446744073709551615]`                 |
+| FLOAT32 | A number that can be represented as a 32-bit IEEE floating point number   |
+| FLOAT64 | A number that can be represented as a 64-bit IEEE floating point number   |
+| BOOLEAN | True or false                                                             |
+| STRING  | A sequence of characters                                                  |
+| ENUM    | An enumerated type                                                        |
+
+Floating-point properties (`FLOAT32` and `FLOAT64`) must not include values `NaN`, `+Infinity`, or `-Infinity`.
+
+[Enum properties](#enums) are denoted by `ENUM`. An enum property must additionally provide the ID of the specific enum it uses, referred to as its enum type (`enumType`).
+
+> **Implementation Note:** Developers of authoring tools should be aware that many JSON implementations support only numeric values that can be represented as IEEE-754 double precision floating point numbers. Floating point numbers should be representable as double precision IEEE-754 floats when encoded in JSON. When those numbers represent property values (such as `noData`, `min`, or `max`) having lower precision (e.g. single-precision float or 8-bit or 16-bit integer), the values should be rounded to the same precision in JSON to avoid any potential boundary mismatches. Numeric property values encoded in binary storage are unaffected by these limitations of JSON implementations.
```

Comment:
```suggestion
> **Implementation Note:** Developers of authoring tools should be aware that many JSON implementations support only numeric values that can be represented as IEEE-754 double precision floating point numbers. Floating point numbers should be representable as double precision IEEE-754 floats when encoded in JSON. When those numbers represent property values (such as `noData`, `min`, or `max`) having lower precision (e.g. single-precision float, 8-bit integer or 16-bit integer), the values should be rounded to the same precision in JSON to avoid any potential boundary mismatches. Numeric property values encoded in binary storage are unaffected by these limitations of JSON implementations.
```

Reply:
also i'm not sure what "boundary mismatches" mean here, I would reword this to be more specific.

Reply:
OK – I've edited your suggestion to remove the word "boundary", I think just "mismatches" is likely clear enough here.

Context: The phrase "bounding mismatch" is taken [from the glTF specification](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#accessors-bounds), albeit in a section on min/max bounds where the term "boundary" is defined by context.